examples/grpc-plugin/grpc-server/main.go

```diff
@@ -25,9 +25,6 @@ const PluginName = "myPlugin"
 func main() {
 	grpcPlug := grpc.NewPlugin(
 		grpc.UseHTTP(&rest.DefaultPlugin),
-		grpc.UseConf(grpc.Config{
-			Endpoint: "localhost:9111",
```

Comment:
The `grpc.UseConf` was used as an example here. Perhaps you could enable extended logging here instead of via config file?

Reply:
That is also an option. I followed the [readme ](https://github.com/ligato/cn-infra/blob/master/examples/grpc-plugin/README.md) which gives an option to use the config file, however, it is ignored when used together with `UseConf`.

Reply:
Yes, that is to provide a way to programmatically override config files.

Reply:
It would be a good idea to detect situations the config file was set programmatically and also from outside and inform about it in the log. One may not realize it and then it looks like some weird bug.